ccd-definition/AuthorisationCaseField.json

```diff
@@ -258,13 +258,27 @@
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "dateSubmittedOnly",
```

Comment:
can you make it clear that this ID is for searching / filtering. Only means nothing to me

Reply:
searchAndFilterDate? 
dateForSearchAndFilter?
searchFilterDate?

Reply:
dateSubmitted -> dateAndTimeSubmitted
dateSubmittedOnly -> dateSubmitted